ocs_ci/ocs/constants.py

```diff
@@ -1193,5 +1193,4 @@
 MCG_TESTS_MIN_NB_ENDPOINT_COUNT = 2
 MAX_NB_ENDPOINT_COUNT = 2
 
-OCS_TAINT = "node.ocs.openshift.io/storage"
```

Comment:
This is still used in the [uninstall module](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/uninstall.py#L254).

Reply:
Changed in Uninstall module